test/python/transpiler/test_pulse_efficient_passmanager.py

```diff
@@ -0,0 +1,228 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests pulse-efficient pass manager API"""
+
+import unittest
+
+from qiskit.converters import circuit_to_dag
+
+from qiskit import QuantumCircuit, QuantumRegister
+from qiskit.quantum_info.synthesis.two_qubit_decompose import TwoQubitWeylDecomposition
+from qiskit.compiler import transpile
+from qiskit.test import QiskitTestCase
+from qiskit.test.mock import (
+    FakeParis,
+    FakeAthens,
+)
+
+from math import pi
+import numpy as np
+
+import qiskit.quantum_info as qi
+
+
+def emptycircuit():
+    """Empty circuit"""
+    return QuantumCircuit()
+
+
+class TestPulseEfficientTranspilerPass(QiskitTestCase):
+    """Test the pulse-efficient pass manager"""
+
+    def setUp(self):
+        super().setUp()
+        self.passes = []
+        self.backends = (FakeParis(), FakeAthens())
+
+    def test_empty_circuit(self):
+        """Test empty circuit"""
+
+        circuit = emptycircuit()
+
+        unitary_circuit = qi.Operator(circuit).data
+
+        result = transpile(circuit, self.backends[0], optimization_level="pulse_efficient")
```

Comment:
Related to the [unnecessary `self.backends`](https://github.com/eggerdj/qiskit-terra/pull/4/files#diff-788b815b2a04ffa57a350567b8b117c0f8e963507a76ec620d57d421989d80ebR45)
```suggestion
        result = transpile(circuit, FakeParis(), optimization_level="pulse_efficient")
```